code/modules/mob/living/living.dm

```diff
@@ -654,34 +654,36 @@
 
 	for(var/obj/effect/decal/cleanable/trail_holder/C in loc) //checks for blood splatter already on the floor
 		blood_exists = 1
```

Comment:
`blood_exists = locate(/obj/effect/decal/cleanable/trail_holder) in loc`
locate will return reference to first trail_holder in loc, but it will be sufficient for the case

Reply:
also, I would move this closer to place, where it's used